deep_qa/run.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import logging
+import shutil
+
+import random
+import pyhocon
+import numpy
+
+# pylint: disable=wrong-import-position
+from deep_qa.common.params import Params, replace_none, ConfigurationError
+from deep_qa.common.tee_logger import TeeLogger
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+def prepare_environment(params: Params):
+    seed = params.pop("random_seed", 13370)
+    numpy_seed = params.pop("numpy_seed", 1337)
+    if "keras" in sys.modules:
+        logger.warning("You have already imported Keras in your code. If you are using DeepQA"
+                       "functionality to set random seeds, they will have no effect, as code"
+                       "prior to calling this function will be non-deterministic. We will not"
+                       "the random seed here.")
+        seed = None
+        numpy_seed = None
+    if seed is not None:
+        random.seed(seed)
+    if numpy_seed is not None:
+        numpy.random.seed(numpy_seed)
+
+    from deep_qa.common.checks import log_keras_version_info
+    log_keras_version_info()
+
+
+def train_model(param_path: str, model_class=None):
```

Comment:
I actually think this should stay as `run_model`, because it trains _and_ evaluates a model, depending on what's in the parameters you're passing.  We should also note this after the first sentence of the docstring.